src/Data/RFC5322.hs

```diff
@@ -179,6 +186,20 @@ data Mailbox =
              AddrSpec
     deriving (Show,Eq)
 
+renderMailboxes :: [Mailbox] -> B.ByteString
+renderMailboxes xs = B.intercalate ", " $ renderMailbox <$> xs
+
+renderMailbox :: Mailbox -> B.ByteString
+renderMailbox (Mailbox n a) =
+    let name =
+            maybe
+                mempty
+                (\x ->
+                      mconcat ["\"", Builder.byteString $ T.encodeUtf8 x, "\" "])
```

Comment:
Yes! It seems I also want to use this for other header optics ...

Reply:
Oof, just realised that the rendering of the display name breaks round tripping. I suspect a problem with the parsing, since the rendering of an encoded word looks correct.

Reply:
In case, that's what I currently have:
```Haskell
renderDisplayName :: T.Text -> Builder.Builder
renderDisplayName x =
    mconcat
        [ "\""
        , Builder.byteString $
          serialiseEncodedWord $
          EncodedWord "utf8" Nothing "Q" (T.encodeUtf8 x)
        , "\" "]
```

Reply:
what's the discrepancy that occurs in the roundtrip?  What does the encoded-word look like?

Reply:
@romanofski can you push a WIP commit?

Reply:
Pushed. Latest HEAD of this pull request/branch (769da44e9102af8941596c97cafc07ace94926bd)